src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -167,9 +167,13 @@
 	// Google connection property names
 	public static final String GGB_SERVER_NAME = "ggb.server";
 	public static final String GGB_GOOGLE_GROUP_DOMAIN = "ggb.google.group.domain";	
+	
```

Comment:
Minor point, but it would be good to group all the GGB values together rather than separate them.